app/main/views/auth.py

```diff
@@ -139,3 +192,35 @@ def select_dept():
         return redirect_to_broker(IDP_OF_LAST_RESORT)
 
     return render_template('views/auth/select_dept.html', form=form)
+
+
+@main.route('/_search-dept')
+def _search_dept():
+    search_term = request.args.get('search_term', 0, type=str)
+    return jsonify([
+        {'id': 'GDS', 'descr': 'Government Digital Services'},
+        {'id': 'CO', 'descr': 'Cabinet Office'}
+    ])
```

Comment:
this code will probably be rewritten when I work properly on the search dept card